torch/_export/serde/schema.py

```diff
@@ -72,7 +72,7 @@ class Device:
 
 @dataclass
 class SymInt(_Union):
-    as_symbol: str
+    as_symbol: Tuple[str, int]
```

Comment:
Why we don't go with the side table as discussed before? Are you going to change this in a later PR?